worlds/kdl3/gifting.py

```diff
@@ -30,14 +33,14 @@ async def pop_object(ctx, key: str, value: str):
     ])
 
 
-async def initialize_giftboxes(ctx, giftbox_key: str, motherbox_key: str, is_open: bool):
+async def initialize_giftboxes(ctx: "SNIContext", giftbox_key: str, motherbox_key: str, is_open: bool) -> None:
     ctx.set_notify(motherbox_key, giftbox_key)
     await update_object(ctx, f"Giftboxes;{ctx.team}", {f"{ctx.slot}":
-        {
-            "IsOpen": is_open,
-            **kdl3_gifting_options
-        }})
-    ctx.gifting = is_open
+                                                       {
+                                                           "IsOpen": is_open,
+                                                           **kdl3_gifting_options
+                                                       }})
+    ctx.client_handler.gifting = is_open
```

Comment:
I just hope this is github displaying stuff weirdly

Reply:
For some reason Pycharm thinks this is the correct whitespace, and I've decided to just stop trying to fight it.